### PR TITLE
Minor fixes

### PR DESCRIPTION
**Pull request by taggrx**

### CODE REVIEW

**Comment by taggrx** [comment]
@nathanosdev check the new `godmode` method compiled for dev only.

File: src/backend/lib.rs [hunk 1]
```diff
@@ -97,47 +97,69 @@ fn prod_release() -> bool {
...
-        }
-        let offset = page * BACKUP_PAGE_SIZE as u64;
-        let current_size = ic_cdk::api::stable::stable64_size();
-        let needed_size = ((offset + buffer.len() as u64) >> 16) + 1;
-        let delta = needed_size.saturating_sub(current_size);
-        if delta > 0 {
-            api::stable::stable64_grow(delta).unwrap_or_else(|_| panic!("couldn't grow memory"));
+    #[update]
+    // Promotes any user to a stalwart with 20k tokens.
+    async fn godmode(user_id: UserId) {
```
Comment (nathanosdev) [inline src/backend/lib.rs:113]: bahaha I love the name 😂

**Review by nathanosdev — DISMISSED** [review]

File: src/backend/lib.rs [hunk 2]
```diff
@@ -97,47 +97,69 @@ fn prod_release() -> bool {
...
+            user.stalwart = true;
+            user.change_karma(CONFIG.trusted_user_min_karma, "test");
+            user.apply_rewards();
+            let principal = user.principal.clone();
+            token::mint(state, account(principal), CONFIG.max_funding_amount);
+        })
+    }
+
+    #[update]
+    // Sets a local cansiter as a bucket if it was created previously.
```
Comment (chmllr) [inline src/backend/lib.rs:126]:
```suggestion
    // Sets a local canister as a bucket if it was created previously.
```

**Review by chmllr — DISMISSED** [review]

File: src/backend/lib.rs (hunk 2 above)
Comment (taggrx) [reply src/backend/lib.rs:126]: Thanks a lot!

**Pull request merged**